doc/sphinx/source/community/release_strategy/release_strategy.rst

```diff
@@ -54,21 +54,27 @@ With the following release schedule, we strive to have three releases per year a
 Upcoming releases
 ^^^^^^^^^^^^^^^^^
 
-- 2.10.0 (Release Manager: `Klaus Zimmermann`_)
+- 2.11.0 (Release Manager: TBD)
 
-+------------+--------------------------+
-| 2023-10-02 |ESMValCore feature freeze |
-+------------+--------------------------+
-| 2023-10-09 |ESMValCore release        |
-+------------+--------------------------+
-| 2023-10-16 |ESMValTool feature freeze |
-+------------+--------------------------+
-| 2023-10-23 |ESMValTool release        |
-+------------+--------------------------+
+Planned for February or March 2024
 
 Past releases
 ^^^^^^^^^^^^^
 
+- 2.10.0 (Release Manager: `Klaus Zimmermann`_)
+
++------------+------------+---------------------------------------------------------------------------------------------+---------------------------------------+
+|  Planned   |    Done    |                                            Event                                            |             Changelog                 |
++============+============+=============================================================================================+=======================================+
+| 2023-10-02 |            |                                  ESMValCore Feature Freeze                                  |                                       |
++------------+------------+---------------------------------------------------------------------------------------------+---------------------------------------+
+| 2023-10-09 | 2023-12-19 | `ESMValCore Release 2.10.0 <https://github.com/ESMValGroup/ESMValCore/releases/tag/v2.10.0>`_ | :ref:`esmvalcore:changelog-v2-10-0` |
++------------+------------+---------------------------------------------------------------------------------------------+---------------------------------------+
+| 2023-10-16 |            |                                  ESMValTool Feature Freeze                                  |                                       |
++------------+------------+---------------------------------------------------------------------------------------------+---------------------------------------+
+| 2023-10-16 | 2023-12-20 | `ESMValTool Release 2.10.0 <https://github.com/ESMValGroup/ESMValTool/releases/tag/v2.10.0>`_ |      :ref:`changelog-v2-10-0`       |
++------------+------------+---------------------------------------------------------------------------------------------+---------------------------------------+
```

Comment:
```suggestion
+------------+------------+-----------------------------------------------------------------------------------------------+---------------------------------------+
|  Planned   |    Done    |                                            Event                                              |             Changelog                 |
+============+============+===============================================================================================+=======================================+
| 2023-10-02 |            |                                  ESMValCore Feature Freeze                                    |                                       |
+------------+------------+-----------------------------------------------------------------------------------------------+---------------------------------------+
| 2023-10-09 | 2023-12-19 | `ESMValCore Release 2.10.0 <https://github.com/ESMValGroup/ESMValCore/releases/tag/v2.10.0>`_ | :ref:`esmvalcore:changelog-v2-10-0` |
+------------+------------+-----------------------------------------------------------------------------------------------+---------------------------------------+
| 2023-10-16 |            |                                  ESMValTool Feature Freeze                                    |                                       |
+------------+------------+-----------------------------------------------------------------------------------------------+---------------------------------------+
| 2023-10-16 | 2023-12-20 | `ESMValTool Release 2.10.0 <https://github.com/ESMValGroup/ESMValTool/releases/tag/v2.10.0>`_ |      :ref:`changelog-v2-10-0`       |
+------------+------------+-----------------------------------------------------------------------------------------------+---------------------------------------+
```

Reply:
This is not rendered properly (I am not quite sure if my changes are correcty, please double-check)